x-pack/plugins/security_solution/public/detection_engine/rule_exceptions/utils/helpers.tsx

```diff
@@ -870,3 +881,102 @@ export const enrichSharedExceptions = (
     });
   });
 };
+
+/**
+ * Creates new Rule exception item, fills it with
+ * new condition ANDED together from the alert highlighted fields
```

Comment:
This function is technically more general right? It can be passed any conditions to create a new item with?

```suggestion
 * Creates new Rule exception item with passed in entries
```

Reply:
Right, still in refactoring phase 😄